doc/changelog.rst

```diff
@@ -25,7 +26,17 @@ Bug fixes
 - Fixed a bug where :meth:`~pymongo.collection.Collection.estimated_document_count`
   would fail with a "CommandNotSupportedOnView" error on views (`PYTHON-2885`_).
 - Fixed a bug where invalid UTF-8 strings could be passed as patterns for :class:`~bson.regex.Regex`
-  objects (`PYTHON-3048`_). :func:`bson.encode` now correctly raises :class:`bson.errors.InvalidStringData`.
+  objects. :func:`bson.encode` now correctly raises :class:`bson.errors.InvalidStringData` (`PYTHON-3048`_).
+- Fixed a bug that caused "connection pool paused" errors in the child process after fork (`PYTHON-3257`_).
```

Comment:
One line above you refer to an error using the `:class:` syntax, but here you use the error message itself. Is this to make it easier to google or is it a typo?

Reply:
updated to `AutoReconnect("connection pool paused")`